_posts/blog/2016-07-07-reactive_mindset_burgers.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: "Building a Reactive Mindset"
+description: Building a Reactive Mindset with the help of Donald Duck and his nephews.
+modified:
+categories: blog
+author: florina_muntenescu
+excerpt: What do Donald Duck and reactive programming have in common? Burgers! Check out why.
+tags: [Android, RxJava]
+date: 2016-07-16T15:39:55-04:00
+---
+Here's another way of explaining reactive programming: this time with Donald Duck and his nephews.
+
+## Donald Duck and His Little Helpers
+
+Donald Duck decided to switch from cinema to the food industry and open up a burger joint. But, he's not doing this alone; his three nephews - Huey, Dewey and Louie - are helping him. Each of them is doing exactly one thing:
+
+* Huey gives Donald Duck buns.
+* Dewey gives Donald Duck pieces of raw meat
+* Louie gives Donald slices of tomatoes
+
+<center>
+<picture>
+	<a href="/images/blog/reactive_mindset_burgers/nephews.png"><img src="/images/blog/reactive_mindset_burgers/nephews.png" alt="Huey, Dewey and Louie" width="350"></a>
+	<figcaption>Huey, Dewey and Louie</figcaption>
+</picture>
+</center>
+
+As soon as the restaurant opens, the nephews start working, continuously supplying Donald with buns, meat and tomato slices.
+
+With every new piece of meat that Donald gets, he first needs to make sure that the meat hasn't gone bad, and use only the fresh meat for his burgers. Then, every piece of fresh meat needs to be cooked.
+
+<center>
+<picture>
+	<a href="/images/blog/reactive_mindset_burgers/filter_map_meat.png"><img src="/images/blog/reactive_mindset_burgers/filter_map_meat.png" alt="filter and map" width="350"></a>
+	<figcaption>Filter the meat pieces and cook them</figcaption>
+</picture>
+</center>
+
+The next step is to make the burger. Donald needs to take the first bun given by Huey, together with the first piece of cooked meat and with the first slice of tomato given by Louie and then "zip" together the first burger.
+
+For the second burger, he will use the second bun from Huey, the second piece of cooked meat and the second slice of tomato.The time took by each of the nephews to do his part is different. Louie will be providing tomato slices faster than Dewey can cook the meat. As a result, Donald has to wait for it to be ready, in order to make the burger. Because he doesn't want to throw away any tomato slices, he will always use the first one on the row, given by Louie.
```

Comment:
Instead of "The time took by each of the nephews to do his part is different" maybe "The processing time of each nephew is different"

Either way there is a space missing in the beginning